src/ccs811.cpp

```diff
@@ -320,6 +321,53 @@ bool CCS811::set_envdata(uint16_t t, uint16_t h) {
   return ok;
 }
 
+// Writes temperature and relativeHumidity (with temperature in C and humidity as percent) to ENV_DATA . Returns false on I2C problems.
+bool CCS811::set_envdataDHT(float temperature, float relativeHumidity) {
+  // Based on: https://github.com/sparkfun/SparkFun_CCS811_Arduino_Library/blob/master/src/SparkFunCCS811.cpp
+  	//Check for invalid temperatures
+	temperature = max(-25,min(temperature,50));
+	
+	//Check for invalid humidity
+	relativeHumidity = max(0,min(relativeHumidity,100));
```

Comment:
Unfortunately, this was not compiling on my side. Maybe I had wired settings in the compiler. But changing that to the following code worked for me:

```suggestion
    //Check for invalid temperatures
    temperature = max(-25.0f,min(temperature,50.0f));

    //Check for invalid humidity
    relativeHumidity = max(0.0f,min(relativeHumidity,100.0f));
```